tensorhive/models/RestrictionSchedule.py

```diff
@@ -65,11 +86,19 @@ def is_valid_schedule_expression(schedule_expression):
     def as_dict(self):
         return {
             'id': self.id,
-            'schedule_days': self.schedule_days,
-            'hour_start': self.hour_start,
-            'hour_end': self.hour_end
+            'scheduleDays': [day.to_str() for day in self.parse_schedule_string(self.schedule_days)],
+            'hourStart': self.hour_start.strftime('%H:%M'),
+            'hourEnd': self.hour_end.strftime('%H:%M')
         }
 
+    @staticmethod
+    def parse_schedule_string(schedule: str) -> List[Weekday]:
+        return [Weekday(int(day)) for day in sorted(schedule)]
+
+    @staticmethod
+    def parse_schedule_list(schedule: List[Weekday]) -> str:
+        return ''.join((sorted(''.join([str(day.value) for day in schedule]))))
```

Comment:
Why do we need to join the list before sorting it? Wouldn't the following code work in the same way?

```python
return ''.join(sorted([str(day.value) for day in schedule]))
```

Reply:
Yup, it would. Don't know what happened here, probably a result of earlier versions of this method in my mind. Fixed!